src/GovUk.Education.ExploreEducationStatistics.Data.Services/ReleaseSubjectService.cs

```diff
@@ -0,0 +1,80 @@
+#nullable enable
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Data.Model;
+using GovUk.Education.ExploreEducationStatistics.Data.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Data.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Services;
+
+public class ReleaseSubjectService : IReleaseSubjectService
+{
+    private readonly StatisticsDbContext _statisticsDbContext;
+    private readonly ContentDbContext _contentDbContext;
+
+    public ReleaseSubjectService(
+        StatisticsDbContext statisticsDbContext,
+        ContentDbContext contentDbContext)
+    {
+        _statisticsDbContext = statisticsDbContext;
+        _contentDbContext = contentDbContext;
+    }
+
+    public async Task<Either<ActionResult, ReleaseSubject>> CheckReleaseSubjectExists(Guid subjectId)
+    {
+        return await CheckReleaseSubjectExists(subjectId, null);
+    }
+
+    public async Task<Either<ActionResult, ReleaseSubject>> CheckReleaseSubjectExists(Guid subjectId, Guid? releaseId)
```

Comment:
Could these be combined? `CheckReleaseSubjectExists(Guid subjectId, Guid? releaseId = null)`?